### PR TITLE
Email Action changes and vulnerability security alerts Fix

### PR DESCRIPTION
**Pull request by redhatHameed**
Added email Action changes.

Signed-off-by: Abdul Hameed <ahameed@redhat.com>

### CODE REVIEW

File: fisalert-service/README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-In this example we will use two containers, one container to run as a ActiveMQ broker, and another as a client to the broker, where the Camel routes is running.
 
-### Backgorund
 
+### Background
 
-This demo is a simple service which sent email alert and exposes an API endpoints,to send a alert by recieving input as a jason post request parameters. These content is later sent to a messaging broker, and then picked up by a bean, which simply process email paramets and returns a result of the email status. 
+
+This service sent an email alert and exposes an API endpoints, by receiving input as a JASON post request parameters. These content is later sent to a messaging broker, and then picked up by a bean, which simply process email parameters and returns a result of the email status.
```
Comment (bmozaffa) [inline fisalert-service/README.md]: This service **_sends_** an email alert and exposes an API endpoints, by receiving input as **_JSON_**. **_This_** content is later sent to a messaging broker, and then picked up by a bean, which simply **_processes_** email parameters and returns a result of the email status.

File: fisalert-service/README.md [hunk 2]
```diff
@@ -25,6 +25,32 @@ The example can be run locally using the following Maven goal:
...
+  "phone": "1234567",
+  "email": "abc@gmail.com,
+  "description": "test alert service"
+}
+
+
+
+Email Conifg:
+
+ - Update src/main/resources/ with your from email addresses.
```
Comment (bmozaffa) [inline fisalert-service/README.md]: with your email addresses?

File: fisalert-service/README.md (hunk 1 above)
Comment (redhatHameed) [reply fisalert-service/README.md]: change done.

File: fisalert-service/README.md (hunk 2 above)
Comment (redhatHameed) [reply fisalert-service/README.md]: sentence renamed.

**Pull request merged**